src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -39,7 +35,7 @@ Notifications are sent to every notification channel on the policy when an incid
 
 ## Incident preference options [#preference-options]
 
-Each option for incident preference has its own advantages. You can select any option for any policy.
+Each option for incident preference has its own advantages. You can select any preference for any policy, so no policy has to have the same incident preference.
```

Comment:
This feels a bit circular, especially the second sentence. Perhaps this can be simplified a bit.

Also, I think I prefer "incident preference options". 

Basically, you can have the same preferences for all your policies or you can use different preferences for each policy, yeah?